libdnf.spec

```diff
@@ -154,6 +155,7 @@ popd
 %{_libdir}/pkgconfig/%{name}.pc
 %{_includedir}/%{name}/
 %{_datadir}/gir-1.0/Dnf-*.gir
+%{_datadir}/vala/vapi/
```

Comment:
Could you guys be a little bit less picky, please? The way the original patch does it is a perfectly valid way to list files in rpm. RPM includes directories recursively if you list %{_datadir}/vala/vapi/ and the suggested change doesn't change the produced binary rpm either way. It's just scaring off possible contributors if you nitpick things to death.